src/lib/Behat/TestFiles/binary1.txt

```diff
@@ -0,0 +1,85 @@
+Feature: Content fields setting and editing
```

Comment:
Is a text file a good enough binary file? 🙂  What is the objective here?

Reply:
Mainly the size of the file - `txt` is not an image, neither a video and at the same time it's easy to prepare a small text file, unlike some encoded formats like `exe` or `dmg`.